src/amp-story-player.js

```diff
@@ -193,12 +234,100 @@ export class AmpStoryPlayer {
       return;
     }
 
-    // TODO(#26308): Layout all child iframes.
-    this.layoutIframe_(this.stories_[0], this.iframes_[0]);
+    // Display first story.
+    const firstStory = this.stories_[0];
+    const iframeIdx = firstStory[IFRAME_IDX];
+    const storyIframe = this.iframes_[iframeIdx];
+    this.layoutIframe_(firstStory, storyIframe);
+    this.displayStory_(iframeIdx);
+
+    // Pre-render next stories.
+    for (let idx = 1; idx < this.stories_.length && idx < MAX_IFRAMES; idx++) {
```

Comment:
Would be cool to start at `i = 0` and have a parameter in `layoutIframe_` to prerender it or not. This way we could merge this block with the one above.
Eg: `this.layoutIframe_(story, iframe, i !== 0 /** prerender */);` with the `prerender` boolean adding a `visibilitystate=prerender` parameter (cf other comment about prerendering)